server/ai/configuration.go

```diff
@@ -12,6 +12,7 @@ type ServiceConfig struct {
 	TokenLimit              int    `json:"tokenLimit"`
 	StreamingTimeoutSeconds int    `json:"streamingTimeoutSeconds"`
 	SendUserID              bool   `json:"sendUserID"`
+	MaxTokens               int    `json:"maxTokens"`
```

Comment:
Maybe we should make these names more clear. What is the difference between MaxTokens and TokenLimit? 

Maybe `InputTokenLimit` and `OutputTokenLimit`? What do you think? 
Then we can update everywhere else to be consistent with that wording.

Reply:
Are you just wanting to rename the struct fields and leaving the json field names the same? Renaming the JSON field for `tokenLimit` would require a migration.

I have to be honest, I'm having a hard time understanding what `TokenLimit` is but from what I can see is `TokenLimit` the `InputTokenLimit` and `MaxTokens` is the `OutputTokenLimit`? I do think both these fields need a description underneath them in system console

Reply:
@BenCookie95 yes let's rename the variables for clarity. We can leave the json for now so we don't have to do a migration. Add a comment explaining. 

Yes you have the mapping correct, the names are quite poor which is why I think we should fix them.

Reply:
Done, https://github.com/mattermost/mattermost-plugin-ai/pull/276/commits/061adc828924443f6dd2c2856215a568016cc284 for the name changes